omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -244,6 +246,7 @@ def generate_hash
       results["private_chef"]["oc-chef-pedant"] = PrivateChef["oc_chef_pedant"]
       results["private_chef"]["notification_email"] = PrivateChef["notification_email"]
       results["private_chef"]["from_email"] = PrivateChef["from_email"]
+      results["private_chef"]["fips_enabled"] = PrivateChef["fips"] || PrivateChef["fips_enabled"]
```

Comment:
Our Chef configuration expects `fips true` so I wanted to provide the same interface to all our products that are going to allow fips enabling at runtime. But because Chef Server internally references `fips_enabled` I thought we could fall back to that.

The more I think about this though, the worse that is. Lets define the interface (`fips true`) and just stick to that.